lib/fabrik/DirectorManager.js

```diff
@@ -545,7 +530,7 @@ class DirectorManager extends BaseManager {
     const backupStartedAt = new Date().toISOString();
     const data = _
       .chain(opts)
-      .pick('service_id', 'plan_id', 'instance_guid', 'organization_guid', 'space_guid', 'username')
+      .pick('service_id', 'plan_id', 'organization_guid', 'space_guid', 'instance_guid', 'username')
```

Comment:
Do you want to drop `space_guid` in the above?

Reply:
Removed